storage-manager/src/smls.cpp

```diff
@@ -80,7 +82,11 @@ void lsOffline(const char *path)
             }
             else
                 cout.width(15);
-            cout << right << _stat.st_size << left << " " << entry << endl;
+
+            struct tm *my_tm = localtime(&_stat.st_mtim.tv_sec);
```

Comment:
Not sure if it matters, but this isn't thread safe

Reply:
it's a single-threaded tool